### PR TITLE
feat: Fixed footer. Progress on event list

### PR DESCRIPTION
**Pull request by TheBeege**

### CODE REVIEW

**Comment by TheBeege** [comment]
I haven't Tailwind-ified everything, but we can look into that afterwards.

I have serious reservations about Tailwind. The purpose of HTML and CSS being separate is to separate the semantics of the page structure (HTML) from the rendering of that structure (CSS). Tailwind just throws that all away. Engineers also need to learn all the different Tailwind classes in addition to normal CSS attributes. I'm not sure I see the value in this.

File: src/routes/events/+page.ts [hunk 1]
```diff
@@ -1,14 +1,51 @@
...
-		events: [
-			{
-				id: 1,
-				eventTitle: 'test',
-				eventDate: new Date(),
-				eventTime: new Date()
-			}
-		]
+export async function load({ fetch }: LoadEvent): Promise<EventListData | null> {
+	const url = 'http://localhost:8000/events';
```
Comment (Casamatechs) [inline src/routes/events/+page.ts]: I would add from the beginning an env variable for the API endpoint so we don't have to care about it in the future as the codebase grows

File: src/routes/events/+page.svelte [hunk 2]
```diff
@@ -4,10 +4,10 @@
 	export let data;
 </script>
 
-<h1>Events</h1>
+<h1>The Latest and Greatest CodeSeoul Events!</h1>
 
 <div>
-	{#if data.events.length === 0}
+	{#if data.events && data.events.length === 0}
```
Comment (Casamatechs) [inline src/routes/events/+page.svelte]: I think it'd be better to set the condition as `{#if !data.events || data.events.length === 0}` so we can also control the case when events doesn't exist inside data (I'm not sure if this is a possibility)

**Comment by Casamatechs** [comment]
> I haven't Tailwind-ified everything, but we can look into that afterwards.
> 
> I have serious reservations about Tailwind. The purpose of HTML and CSS being separate is to separate the semantics of the page structure (HTML) from the rendering of that structure (CSS). Tailwind just throws that all away. Engineers also need to learn all the different Tailwind classes in addition to normal CSS attributes. I'm not sure I see the value in this.

I agree that Tailwind throws away the separation between semantics, but at the same time I find it really helpful to keep track of what's going on with the layout, specially when you're not familiar with the code. With Svelte it may not make that much sense as we have isolated CSS classes generated on each component, but I think it doesn't hurt to have both options available.

**Review by Casamatechs — COMMENTED** [review]
I would suggest two small changes to the PR, you can find comments for both of them

File: src/routes/events/+page.ts (hunk 1 above)
Comment (TheBeege) [reply src/routes/events/+page.ts]: Yeah, I was being lazy, but good idea to get it handled now

File: src/routes/events/+page.svelte (hunk 2 above)
Comment (TheBeege) [reply src/routes/events/+page.svelte]: Ah! Good catch

**Review by Casamatechs — APPROVED** [review]
LGTM!

**Pull request merged**